### DISCUSSION TITLE
Documentation link not working

### DISCUSSION THREAD

**Discussion by MaxSustainableSolutions** [GeometryGym/GeometryGymIFC#92] [General]
Hi i tried to find the documentation of these classes, however the documentation link (http://www.buildingsmart-tech.org/ifc/IFC4/Add1/html/ ) does not work. Where can i find it?

**Comment by jmirtsch** [comment]
Where did you get the link from?  I can see if I can fix it.  buildingSMART changed the domains and closed the old one, so there is no forwarding.  You can find the documentation at eg https://standards.buildingsmart.org/IFC/RELEASE/IFC4/ADD2_TC1/HTML/
or other versions at https://technical.buildingsmart.org/standards/ifc/ifc-schema-specifications/

**Comment by MaxSustainableSolutions** [comment]
Thanks, it is still in the readMe file: https://github.com/GeometryGym/GeometryGymIFC

**Comment by jmirtsch** [comment]
Thanks for pointing that out, seems I haven't updated that text for some time.  I've just made some improvements.
